web/app.js

```diff
@@ -1851,6 +1857,11 @@ const PDFViewerApplication = {
     this.pdfPresentationMode?.request();
   },
 
+  requestFreeTextMode() {
+    AnnotationEditorLayerBuilder.setEditorType("FreeText");
```

Comment:
(While the viewer only really supports being used standalone, it seems a bit weird that the editor-mode is global. However, I guess that this considerably simplifies the overall implementation?)